Vagrantfile

```diff
@@ -154,6 +154,47 @@ Vagrant.configure(2) do |config|
     done
   SHELL
 
+  # Set up Istio.
+  config.vm.provision :shell, privileged: false, inline: <<-'SHELL'
+    workspace=$PWD
+    # Download the Istio release.
+    version="1.0.5"
+    curl -L https://git.io/getLatestIstio | ISTIO_VERSION=${version} sh
+    export PATH=${workspace}/istio-${version}/bin:$PATH
+    # Create a service account for Tiller.
+    kubectl apply -f ${workspace}/istio-${version}/install/kubernetes/helm/helm-service-account.yaml
```

Comment:
Per the discussion below, there will be another separate issue refine `helm init` right?

Reply:
I think we could do in this PR:

1. Create the Helm service account using a file checked into the repo rather than the one provided by Istio.
2. Initialize Helm with the service account.
3. Then lines 164-188 won't be needed.